cmd/kogito/command/install/kogito_operator.go

```diff
@@ -107,18 +102,29 @@ func (i *installKogitoOperatorCommand) Exec(cmd *cobra.Command, args []string) e
 		return err
 	}
 
-	install := shared.ServicesInstallationBuilder(i.Client, i.flags.namespace).InstallOperator(true, i.flags.image, i.flags.force, shared.KogitoChannelType(i.flags.Channel))
-	if i.flags.installDataIndex || i.flags.installAllServices {
+	install := shared.
+		ServicesInstallationBuilder(i.Client, i.flags.namespace).
+		InstallOperator(true, i.flags.image, i.flags.force, shared.KogitoChannelType(i.flags.Channel))
+
+	if i.flags.installDataIndex || i.flags.installJobsService || i.flags.installManagementConsole {
+		persistenceInfra := shared.GetDefaultPersistenceInfra(i.flags.namespace)
+		install.InstallInfraService(persistenceInfra)
```

Comment:
For Jobs service we don't need persistence or messaging by default. Also AFAIK the Management console requires Data index only, the persistence and messaging are transitive dependencies in that case.

Reply:
Ah yes, removed the jobsService and mgmtConsole from this condition